packages/app-desktop/gui/NoteEditor/utils/useFormNote.ts

```diff
@@ -58,12 +59,15 @@ function resourceInfosChanged(a: ResourceInfos, b: ResourceInfos): boolean {
 	return false;
 }
 
+const getCompareKey = (a: FormNote) => `${a.body}|${a.id}|${a.title}|${a.parent_id}|${a.is_todo}`;
+const isNoteEqual = (a: FormNote, b: FormNote) => getCompareKey(a) === getCompareKey(b);
+const isNewer = (a: FormNote, b: any) => a.user_updated_time > b.update_time;
```

Comment:
Did you use "any" because it didn't compile? Because, yes it wouldn't since update_time doesn't exist. Also please only compare updated_time since user_updated_time can be set to anything by the user.

Reply:
- I used `any` because everywhere else in this repository you also used `any`  for the note.

- `FormNote` does not have the `updated_time` property ([see type](https://github.com/laurent22/joplin/blob/0b01b5b0ef7a2e9
ac9e96a53b85bdf44e14d2f18/packages/app-desktop/gui/NoteEditor/utils/types.ts#L81))…

This is the content of the FormNote object:
<img width="574" alt="Screen Shot 2022-04-02 at 19 05 57" src="https://user-images.githubusercontent.com/12467511/161393580-8f547b3c-886a-4ba4-9781-f571635e04d3.png">